lib/mix/tasks/tailwind.install.ex

```diff
@@ -38,11 +38,22 @@ defmodule Mix.Tasks.Tailwind.Install do
           Tailwind.install()
         end
 
+      {opts, [base_url]} ->
+        if opts[:runtime_config], do: Mix.Task.run("app.config")
+
+        if opts[:if_missing] && latest_version?() do
+          :ok
+        else
+          Tailwind.install(base_url)
+        end
+
       {_, _} ->
         Mix.raise("""
         Invalid arguments to tailwind.install, expected one of:
 
             mix tailwind.install
+            mix tailwind.install https://github.com/tailwindlabs/tailwindcss/releases/download/
+            mix tailwind.install https://github.com/tailwindlabs/tailwindcss/releases/download/v3.2.4/tailwindcss-macos-x64
```

Comment:
According to my comment, this would become:

```suggestion
            mix tailwind.install https://github.com/tailwindlabs/tailwindcss/releases/download/v$version/tailwindcss-$target
```